addons/udes_security/models/res_users.py

```diff
@@ -99,3 +99,21 @@ def _check_is_admin(self, values, group):
                     )
                     % group.name
                 )
+
+    @api.model_cr_context
+    def _auth_timeout_enabled(self):
+        """ Pluggable method to check if session timeout is enabled """
+        auth_timeout_enabled_parameter = self.env["ir.config_parameter"].get_param(
```

Comment:
Do we want to use `IrConfigParameter` instead?

Reply:
Yep, changed. :)